ansible_builder/cli.py

```diff
@@ -98,7 +111,7 @@ def parse_args(args=sys.argv[1:]):
                                       default=constants.default_tag,
                                       help='The name for the container image being built (default: %(default)s)')
 
-    for p in [build_command_parser]:
+    for p in [create_command_parser, build_command_parser]:
```

Comment:
Some time has passed since this look was last functional, and not all of these options look like they should apply to the `create` command.

 - `--container-runtime` would seem to have no effect on `ansible-builder create`, so it would be best to exclusively add it to the build_command_parser
 - `--build-arg` also only gets used on `podman build`, so it's irrelevant for create, so same goes for this

I think the rest of them look right.

Reply:
Done, now the `--help` for `anisble-builder build` shows the following options:

![Screenshot from 2021-06-03 11-50-08](https://user-images.githubusercontent.com/28930622/120674807-785afc80-c462-11eb-9158-2fabc854e7ee.png)


and for `create` it shows:

![Screenshot from 2021-06-03 11-49-32](https://user-images.githubusercontent.com/28930622/120674825-7d1fb080-c462-11eb-9c21-bbb9bd45fab4.png)